### PR TITLE
Added a picture of myself and support for multiple images for every team member.

### PR DESCRIPTION
**Pull request by MAMV3x3**
I modified the css so that when including multiple images in the profile div they do not overlap each other, I added media queries so that the design is responsive and I added a photo of myself, leaving the photo of Will and two more spaces for the photos of my fellowmates.
In the `app/templates/index.html` file I I added comments to make it clearer where to modify the images.

Current result:
image

### CODE REVIEW

File: app/static/styles/main.css [hunk 1]
```diff
@@ -52,29 +52,28 @@ body {
 .profile {
     background-color: #1d539f !important;
     text-align: center;
-    height: 320px;
+    height: auto;
     margin-bottom: 50px;
 }
 
 .profile h1 {
-        font-size: 50px;
```
Comment (a-brown-code) [inline app/static/styles/main.css:60]: Just curious, since it seems to have no bearing on the functionality of the code--why the change in whitespace?

File: app/templates/index.html [hunk 2]
```diff
@@ -32,7 +32,14 @@
 
     <div class="profile">
         <div id="profile-picture" class="profile-picture">
-            <img src="./static/img/logo.jpg">
+            <!-- Will's Profile Picture -->
```
Comment (a-brown-code) [inline app/templates/index.html:35]: Do we need to keep Will's? I assumed that was the one we would be replacing.

**Review by a-brown-code — APPROVED** [review]
Looks good to me! Great work 😄

File: app/static/styles/main.css (hunk 1 above)
Comment (MAMV3x3) [reply app/static/styles/main.css:60]: Oh, I'm using Prettier and it changed the indentation automatically

File: app/templates/index.html (hunk 2 above)
Comment (MAMV3x3) [reply app/templates/index.html:35]: In fact I'm not sure, what do you think? should we keep it or remove it? 👀

Comment (a-brown-code) [reply app/templates/index.html:35]: I say we get rid of it to make it a little more our own 🪓

Comment (Pradyuman7) [reply app/templates/index.html:35]: Yes, let's remove it. Your portfolio is yours only 😉

File: app/static/styles/main.css [hunk 3]
```diff
@@ -52,29 +52,28 @@ body {
...
 
 .profile img {
-        border-radius: 50%;
-        max-width: 100%;
-        max-height: 100%;
-        margin-left: auto;
-        margin-right: auto;
-        display: block;        
-        padding: 20px;
+    width: 200px;
```
Comment (Pradyuman7) [inline app/static/styles/main.css:66]: I suppose that using px is fine in this case, but generally `em` is also a good option as em is relative to the parent element

File: app/static/styles/main.css (hunk 1 above)
Comment (Pradyuman7) [reply app/static/styles/main.css:60]: I suggest that you all setup a single kind of environment with such extensions in your IDE so that the indentation and other changes are consistent

File: app/static/styles/main.css [hunk 4]
```diff
@@ -52,29 +52,28 @@ body {
...
-    height: 320px;
+    height: auto;
     margin-bottom: 50px;
 }
 
 .profile h1 {
-        font-size: 50px;
-        font-weight: 700;
-        color: white;
+    font-size: 50px;
```
Comment (Pradyuman7) [inline app/static/styles/main.css:60]: em is generally better for font sizes

File: app/templates/index.html (hunk 2 above)
Comment (MAMV3x3) [reply app/templates/index.html:35]: I'm going to remove Will's photo and add an avatar logo for now while you all upload your photos as well 🙌🏻

**Comment by MAMV3x3** [comment]
I made a new **commit** `(ID: 79890f9ed1131c8e13591317cb36d28e3a5ce6be)` including all the proposed suggestions, I hope you can review it and give me feedback on it!

**Review by Pradyuman7 — APPROVED** [review]
LGTM 🔥
